AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockSafeContentFilterTest.xml

```diff
@@ -35,10 +35,6 @@
         <actionGroup ref="AdminAdobeStockExpandFiltersActionGroup" stepKey="expandFilters"/>
         <actionGroup ref="AdminUserApplySafeSearchFilterActionGroup" stepKey="applySafeSearchFilter"/>
         <grabTextFrom selector="{{AdminAdobeStockSection.recordsFound}}" stepKey="countWithAppliedFilter"/>
-        <assertNotEquals stepKey="checkThatResultCountChangesAfterFilterApplied">
```

Comment:
After closer look to this test I believe that check is necessary, let's revert this change and try to approach it from the modifiers